marthamareal/ridemyway-api-django
# #163859914 Create user model

#### What does this PR do?
- Creates a model for the user with its serializer.
#### Description of Task to be completed?
- create a class for the use model
- create a serializer for the model
- test the model and serializers

#### How should this be manually tested?
* Fetch the branch to you local machine with ` git fetch origin ch-create-user-model-163859914`
* Check out the branch `git checkout ch-create-user-model-163859914`
* Run migrations with `python manage.py runserver`
* Check your database, you will find the user table created
#### Any background context you want to provide?
- Refer to [ModelSerializers](https://www.django-rest-framework.org/api-guide/serializers/#modelserializer) to relate with our serializer implementation.
#### What are the relevant pivotal tracker stories?
[#163859914](https://www.pivotaltracker.com/story/show/163859914)
#### Screenshots(if applicable):
- N/A
